analysis.tex

```diff
@@ -77,177 +68,31 @@ \section{Analysis}
   the protocol $\Pi$ is binding.
 \end{proof}
 
-Recall that a distribution ensemble is \emph{unpredictable} if no
-polynomial-time adversary can guess its output. The cryptographic
-predictability game is illustrated in Algorithm~\ref{alg.predict-game}.
-
-\import{.}{./algorithms/alg.predict-game.tex}
-
-\begin{definition}[Unpredictable distribution]
-  A distribution ensemble $\{X_{\kappa}\}_{\kappa\in\mathbb{N}}$ is
-  \emph{unpredictable} if for all probabilistic polynomial-time adversaries
-  $\mathcal{A}$ there exists a negligible function $\negl$ such that
-  $\Pr[\predict_{\mathcal{A},X}(\kappa) = \true] < \negl$.
-\end{definition}
-
-\begin{lemma}[Negligible unpredictability]\label{lem:negl-unpred}
-  Consider a distribution ensemble $\{X_{\kappa}\}_{\kappa\in\mathbb{N}}$ and
-  a negligible function $\negl$. If
-  $\max_{x \in [X_\kappa]}\Pr_{x^* \gets X_\kappa}[x^* = x] \leq \negl$,
-  then $X$ is unpredictable.
-\end{lemma}
-\begin{proof}
-  Consider a probabilistic polynomial-time adversary $\mathcal{A}$ which
-  predicts $X_\kappa$. The adversary is not given any input beyond $1^\kappa$,
-  hence the distribution of its output is independent from the choice of the
-  challenger. Therefore
-
-  \begin{align*}
-  \Pr[\predict_{\mathcal{A},X}(\kappa) = \true] &=
-  \sum_{x' \in [X]}\Pr_{x \gets X}[\mathcal{A}(\kappa) = x'\land x = x'] =\\
-  \sum_{x' \in [X]}\Pr[\mathcal{A}(\kappa) = x']\Pr_{x \gets X}[x = x']
-  &\leq \negl\sum_{x' \in [X]}\Pr[\mathcal{A}(\kappa) = x']
-  \leq \negl
-  \,.
-  \end{align*}
-\end{proof}
-
-We now posit that no adversary can predict the public key of a secure signature scheme, except with negligible probability.
+We now posit that no adversary can predict the public key of a secure signature scheme, except with negligible probability. We call a distribution \emph{unpredictable} if no
+probabilistic polynomial-time adversary can predict its sampling. We give
+the formal definition, with some of its statistical properties, in
+Appendix~\ref{sec:proofs}.
```

Comment:
this should be a ref to the subsection specific for unpredictable dists., "B.2 Unpredictable distributions"